client/src/hook.rs

```diff
@@ -0,0 +1,42 @@
+use std::io::{BufRead, BufReader, Write};
+use std::os::unix::net::UnixStream;
+use std::thread;
+
+use common::client_daemon::{serialize_message_client_to_daemon, ClientToDaemonMsg};
+use common::notif::notif;
+
+pub fn hook(path: &'static str) {
+    thread::spawn(move || {
+        let mut stream = match UnixStream::connect(path) {
+            Ok(stream) => stream,
+            Err(e) => {
+                notif(
+                    "Client Error",
+                    &format!("Could not serialize packet: {:?}", e),
+                );
+                return;
+            }
+        };
+        let mut to_send = match serialize_message_client_to_daemon(ClientToDaemonMsg::Test(
```

Comment:
Is this the protocol start or just a test for now?

Reply:
Oh i see, this is the Test()

Reply:
Nope just a test, we'll add some hooking logic packets later on.